### PR TITLE
Add blackhole test case, simulating network failure

### PR DESCRIPTION
**Pull request by zuiderkwast**

### CODE REVIEW

File: test/ered_SUITE.erl [hunk 1]
```diff
@@ -213,6 +214,56 @@ t_manual_failover(_) ->
...
+    Pod = get_pod_name_from_port(Port),
+    ClientPid = maps:get({"127.0.0.1", Port}, ered:get_addr_to_client_map(R)),
+    ct:pal("Pausing container: " ++ os:cmd("docker pause " ++ Pod)),
+
+    %% Now when a command times out, the ered_connection process is restarted
+    %% but connect() times out. Ered reports that one is down, thus the cluster
+    %% is not OK. Failover happens. Ered discovers the new master and reports
+    %% that the cluster is OK again.
+    TestPid = self(),
+    ered:command_client_async(ClientPid, [<<"PiNG">>],
```
Comment (bjosv) [inline test/ered_SUITE.erl:242]:
```suggestion
    ered:command_client_async(ClientPid, [<<"PING">>],
```

Comment (bjosv) [reply test/ered_SUITE.erl:242]: just for consistency

**Review by bjosv — COMMENTED** [review]
Nice! So this seems to work as expected then?

**Comment by zuiderkwast** [comment]
> Nice! So this seems to work as expected then?

Yes, well, I whatched what happens and then made the test case accordingly.

I wonder what happens if we put a sleep in the ReplyFun. Maybe its delays ered_client's exit enough to make ered kill itself by killing its own reconnnect loop pid...? Next PR?

File: test/ered_SUITE.erl (hunk 1 above)
Comment (zuiderkwast) [reply test/ered_SUITE.erl:242]: Consistency with what?

**Pull request merged**